templates/Notifications/NotificationTypes/broadcast_notification.html.twig

```diff
@@ -1,15 +1,14 @@
-{% extends 'Notifications/NotificationTypes/baseNotification.html.twig' %}
+{% extends 'Notifications/NotificationTypes/notification_items.html.twig' %}
 
-{% block notification_title%}
-  {{ notification.title }}
-{% endblock %}
 
 {% block notification_leftAvatar %}
-   <div class="notification-avatar text-primary">
-     <i class="fa fa-5x fa-bullhorn"></i>
-   </div>
+  <div class="col-2 col-sm-1 my-auto">
+    <i class="fa broadcast-icon fa-bullhorn"></i>
```

Comment:
please use material.io icon

Reply:
![image](https://user-images.githubusercontent.com/40868718/89381938-381fd600-d6fa-11ea-80ec-339417059f9e.png)

There is no title in the new design. So you can remove it from the admin interface as well please

Reply:
![image](https://user-images.githubusercontent.com/40868718/89382033-61406680-d6fa-11ea-8013-9f532b4fa27c.png)